### PR TITLE
Slave

### PR DESCRIPTION
**Pull request by almong**

### CODE REVIEW

File: views/index.php [hunk 1]
```diff
@@ -7,33 +7,31 @@
         <a type="button" class="btn btn-success">Задать вопрос</a>
 
 	<ul class="cd-faq-categories">
-        <?php
-        foreach ($categorys as $category){
+        <?php foreach ($categorys as $category):
```
Comment (sg4tech) [inline views/index.php:10]: `<?php foreach ($categorys as $category): ?>`

File: views/index.php [hunk 2]
```diff
@@ -7,33 +7,31 @@
...
             if ($category['id'] == 1) {?>
```
Comment (sg4tech) [inline views/index.php:11]: `<?php if ($category['id'] == 1) : ?>`

File: views/index.php [hunk 3]
```diff
@@ -7,33 +7,31 @@
...
     		<li><a href="#<?= $category['name']?>"><?= ucfirst($category['name'])?></a></li>
-        <?php } }?>
+        <?php } endforeach;?>
 	</ul> <!-- cd-faq-categories -->
 
 	<div class="cd-faq-items">
         <a class="btn btn-outline-info" href="question" role="button">Задать вопрос</a>
-        <?php
-            foreach ($categorys as $category){?>
+        <?php foreach ($categorys as $category):?>
```
Comment (sg4tech) [inline views/index.php:20]: [c] categories

File: views/index.php (hunk 1 above)
Comment (almong) [reply views/index.php:10]: done

File: views/index.php (hunk 2 above)
Comment (almong) [reply views/index.php:11]: done

File: views/index.php (hunk 3 above)
Comment (almong) [reply views/index.php:20]: done

**Comment by almong** [comment]
Все исправил

**Pull request merged**